devices/wholeBodyDynamics/WholeBodyDynamicsDevice.h

```diff
@@ -114,210 +114,9 @@ class wholeBodyDynamicsDeviceFilters
- *         <group name="IDYNTREE_SKINDYNLIB_LINKS">
- *             <param name="root_link">(root_link,1,0)</param>
- *             <param name="chest"> (chest,1,2)</param>
- *             <param name="l_upper_arm">(l_upper_arm,3,2)</param>
- *             <param name="l_elbow_1">(l_elbow_1, 3, 4)</param>
- *             <param name="r_upper_arm">(r_upper_arm,4,2)</param>
- *             <param name="r_elbow_1">(r_elbow_1, 4, 4)</param>
- *             <param name="l_lower_leg">(l_lower_leg,5,3)</param>
- *             <param name="l_ankle_1">(l_ankle_1,5,4)</param>
- *             <param name="l_foot">(l_foot_dh_frame,5,5)</param>
- *             <param name="r_lower_leg">(r_lower_leg,6,3)</param>
- *             <param name="r_ankle_1">(r_ankle_1,6,4)</param>
- *             <param name="r_foot">(r_foot_dh_frame,6,5)</param>
- *         </group>
- *
- *         <group name="WBD_OUTPUT_EXTERNAL_WRENCH_PORTS">
- *             <param name="/wholeBodyDynamics/left_leg/cartesianEndEffectorWrench:o">(l_foot,l_sole,root_link)</param>
- *             <param name="/wholeBodyDynamics/right_leg/cartesianEndEffectorWrench:o">(r_foot,r_sole,root_link)</param>
- *         </group>
- *
- *         <action phase="startup" level="15" type="attach">
- *             <paramlist name="networks">
- *                 <!-- motorcontrol and virtual torque sensors -->
- *                 <elem name="left_lower_leg">left_lower_leg_mc</elem>
- *                 <elem name="right_lower_leg">right_lower_leg_mc</elem>
- *                 <elem name="left_upper_leg">left_upper_leg_mc</elem>
- *                 <elem name="right_upper_leg">right_upper_leg_mc</elem>
- *                 <elem name="torso">torso_mc</elem>
- *                 <elem name="right_lower_arm">right_lower_arm_mc</elem>
- *                 <elem name="left_lower_arm">left_lower_arm_mc</elem>
- *                 <elem name="right_upper_arm">right_upper_arm_mc</elem>
- *                 <elem name="left_upper_arm">left_upper_arm_mc</elem>
- *                 <elem name="head">head_mc</elem>
- *                 <!-- imu -->
- *                 <elem name="imu">inertial</elem>
- *                 <!-- ft -->
- *                 <elem name="l_arm_ft_sensor">left_upper_arm_strain</elem>
- *                 <elem name="r_arm_ft_sensor">right_upper_arm_strain</elem>
- *                 <elem name="l_leg_ft_sensor">left_upper_leg_strain</elem>
- *                 <elem name="r_leg_ft_sensor">right_upper_leg_strain</elem>
- *                 <elem name="l_foot_ft_sensor">left_lower_leg_strain</elem>
- *                 <elem name="r_foot_ft_sensor">right_lower_leg_strain</elem>
- *             </paramlist>
- *         </action>
- *
- *         <action phase="shutdown" level="2" type="detach" />
- *
- *     </device>
- * </devices>
- * \endcode
+ * 
+ * For a detailed documentation of the parameters used within this device, please see
+ * robotology/whole-body-estimators/devices/wholeBodyDynamics/README.md
```

Comment:
This reference is confusing, as if you are in a fork you should refer to the README of that fork, I would simple refer to a README in the same directory of this file.

Reply:
Fixed in https://github.com/robotology/whole-body-estimators/pull/84/commits/52a91d13fcc41230bada87da23948da2c8f099f6